custom/enikshay/integrations/nikshay/repeaters.py

```diff
@@ -25,8 +25,9 @@ def get_custom_url(cls, domain):
     def allowed_to_forward(self, episode_case):
         # When case property episode.episode_pending_registration transitions from 'yes' to 'no',
         # and (episode.nikshay_registered != 'true'  or episode.nikshay_id != '')
-        (not episode_case.nikshay_registered and
-         not episode_case.nikshay_id and
+        episode_case_properties = episode_case.dynamic_case_properties()
+        (not episode_case_properties['nikshay_registered'] and
```

Comment:
I think this will fail hard if these properties aren't set. You should use the `.get()` pattern instead.